control-plane/agents/core/src/node/service.rs

```diff
@@ -78,6 +78,16 @@ impl NodeOperations for Service {
         let blockdevices = self.get_block_devices(&req).await?;
         Ok(blockdevices)
     }
+
+    async fn cordon(&self, id: NodeId, label: String) -> Result<Node, ReplyError> {
+        let node = self.cordon(id, label).await?;
```

Comment:
I did this to be in keeping with the rest of the file. The reason we seem to do this currently is because it won't implicitly convert from a `SvcError` to a `ReplyError`.